programs/ssl/ssl_server2.c

```diff
@@ -3781,7 +3781,6 @@ int main(int argc, char *argv[])
             switch (ret) {
                 case MBEDTLS_ERR_SSL_PEER_CLOSE_NOTIFY:
                     mbedtls_printf(" connection was closed gracefully\n");
-                    ret = 0;
                     goto close_notify;
```

Comment:
This is a jump over 300 lines!!

Reply:
Many sample programs, and even more the SSL test programs, could use some restructuring. But it's nowhere near the top of my priorities for technical debt resumption.